src/main/java/com/rewedigital/composer/routing/RoutingConfiguration.java

```diff
@@ -34,16 +36,20 @@ public static RoutingConfiguration fromConfig(final Config config) {
         final String method = config.getString("method");
         final String type = config.getString("type");
         final String target = config.getString("target");
+        final Optional<Duration> ttl = optionalDuration(config, "ttl");
 
-        final Rule<Match> result = Rule.fromUri(path, method, Match.of(target, RouteTypeName.valueOf(type)));
-        LOGGER.info("Registered local route for path={}, method={}, target={}, type={}", path, method,
-            target, type);
+        final Rule<Match> result = Rule.fromUri(path, method, Match.of(target, ttl, RouteTypeName.valueOf(type)));
+        LOGGER.info("Registered local route for path={}, method={}, target={}, type={} with a request ttl={}", path,
+            method,target, type, ttl);
         return result;
-
     }
 
     public List<Rule<Match>> localRules() {
         return localRoutes;
     }
 
+    private static Optional<Duration> optionalDuration(final Config config, final String path) {
```

Comment:
You could use `com.spotify.apollo.environment.ConfigUtil` here instead

Reply:
That utility is private and final

Reply:
On my classpath it is `public final class ConfigUtil `

Reply:
Thanks for remembering me to proof what I think I remember..

Reply:
My fault, I do not really remember why I rolled my own impl.

Reply:
There is no version for Duration, therefore I rolled my own impl.